core/src/services/mod.rs

```diff
@@ -213,7 +213,7 @@ pub use self::redb::Redb;
 #[cfg(feature = "services-tikv")]
 mod tikv;
 #[cfg(feature = "services-tikv")]
-pub use self::tikv::Tikv;
+pub use self::tikv::TikvConfig;
```

Comment:
You may need to use them both.

Reply:
Thanks for the reminder, I'm still familiar with opendal, and I'm sorry for making such a low-level mistake